treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -205,11 +211,13 @@ def _ignore_sensitive_fields(self, table_name, model_values):
         """
         UPSTREAM_DATABASE_URL credentials don't generally have read rights on
         the auth_user table (which contains users' password hashes).
-        Simply setting `classifier = None` won't cause access errors.
+        Simply setting `model.sensitive_field = None` won't cause access errors.
         """
-        if table_name == 'performance_alert':
-            for model in model_values:
+        for model in model_values:
```

Comment:
This isn't too efficient. Most of the time, this will iterate through many, many models, without doing anything.
It should return fast if the table_name isn't 'performance_alert' or 'performance_alert_summary'.

Reply:
I changed the code. This is better, I think.